content/influxdb/v2.0/get-started.md

```diff
@@ -315,6 +315,32 @@ docker run --name influxdb -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4
 ```
 _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._
 
+### Persist data outside the InfluxDB container
+
+1. Create a new directory in which to store your data, and change to that directory.
+   ```sh
+   mkdir my-influxdb-docker-data
+   cd my-influxdb-docker-data/
+   ```
```

Comment:
```suggestion
1. Create a new directory to store your data in and navigate into the directory.
   
   ```sh
   mkdir path/to/influxdb-docker-volume && cd $_
   ```
```

Reply:
`path/to/influxdb-docker-volume` sounds to me like something that should already exist... How about `path/to/influxdb-docker-data-volume`?